packages/host/types/runtime-spike/index.d.ts

```diff
@@ -1,4 +0,0 @@
-import 'ember-concurrency-async';
-import 'ember-concurrency-ts/async';
-
-export {};
```

Comment:
We can leave this file in place with just `export {}`. It's one of the standard ones generated by the ember-cli-typescript app blueprint, and some addons will instruct you to put things in here, so it may be confusing for it not to exist.

Reply:
👍🏻 I’ve restored it, thanks. Does it matter that it’s in `runtime-spike`?

Reply:
yeah, we can probably do a better job with the host app's name. maybe `boxel-host`?

Reply:
since the package name is `@cardstack/host`, maybe that? That’s the pattern in [`@cardstack/web-client`](https://github.com/cardstack/cardstack/blob/main/packages/web-client/types/%40cardstack/web-client/index.d.ts).

Reply:
I’ll just do that for now, if we want to rename all the packages (might make sense for consistency), we can do it separately.